core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
-- name: Set rabbitmq_master fact
+- name: Set facts
   set_fact:
     rabbitmq_master: "{{ groups['rabbitmq'][0] }}"
+    long_names: "{% if specification.rabbitmq_use_longname|bool %}--longnames{% else %}{% endif %}"
```

Comment:
Maybe can be done different way? Maybe with using when, not to write such weird template?

Reply:
Removed it at all as variable should be used.